src/styles/GlobalStyle.js

```diff
@@ -6,17 +6,25 @@ export default createGlobalStyle`
 }
 
 body {
-    font-family: sans-serif;
+    font-family: 'Quicksand', sans-serif;
     font-size: 112.5%;
     margin: 0;
     position: relative;
-
 }
 
 :root {
-    --primary-light: #e6eded;
-    --primary-dark: #2e3332;
-    --primary-dark-transparent: #2e3332BF;
-    --white: #fff;
+    --primary-light: #679FBE;
+    --primary-regular: #296281;
+    --primary-dark: #3C3C3C;
+    --primary-dark-transparent: #3C3C3CBF;
+    --secondary: #B261A0;
+    --white: #FFF;
+    --white-transparent: #FFFFFFF2;
+    --spotify-green: #08DA73;
+    --spotify-green-transparent: #08DA7333;
+    --cta-red: #FF6262;
```

Comment:
Just because I am curious: What is ``cta``?

Reply:
cta stands for call to action :)